### PR TITLE
Update: Python.Python.3.11 version 3.11.4

### LINKED ISSUE #109360: [Update Request]: Python 3.11
### What type of update are you requesting?

A new version of an existing package

### Current Package Identifier

Python.Python.3.11

### Package Version

3.11.4

### Please describe the changes you would like to see

This was reported at Dev Home:
* https://github.com/microsoft/devhome/issues/717

The URL provided by @localden is:
https://www.python.org/ftp/python/3.11.4/python-3.11.4-amd64.exe

### PR DESCRIPTION
**Pull request by Trenly**
﻿- [X] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [X] This PR only modifies one (1) manifest
- [X] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [X] Have you tested your manifest locally with `winget install --manifest <path>`?
- [X] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

* Resolves #109360

 ###### Microsoft Reviewers: codeflow:open?pullrequest=https://github.com/microsoft/winget-pkgs/pull/109368&drop=dogfoodAlpha

### CODE REVIEW

**Comment by denelon** [comment]
1.5 isn't supported yet.

**Comment by denelon** [comment]
I think we should look at adding some kind of label for the schema version. Then we can also include a fabric bot message to explain:

Obsolete: This version is no longer supported
Deprecated: This version is supported, but if you upgrade to {latest} you may get the following benefits:
* a
* b
* c
Not yet supported.

Manifest-Schema-Obsolete "This manifest schema version is no longer supported"
Manifest-Schema-Deprecated "This manifest schema version is not the latest"
Manifest-Schema-Unsupported "This manifest schema version is not yet supported"

**Comment by denelon** [comment]
One label is "easier" and it's only one rule to update...
I'll add that "Manifest-Version-Error" label.

**Comment by denelon** [comment]
Nice!

File: manifests/p/Python/Python/3/11/3.11.4/Python.Python.3.11.locale.zh-CN.yaml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+# Moniker:
+Tags:
+- python
+- python3
+- 编程
+- 编程语言
+- 脚本
+- 语言
+# ReleaseNotes:
+ReleaseNotesUrl: https://docs.python.org/release/3.11.3/whatsnew/changelog.html
```
Comment (mdanish-kh) [inline manifests/p/Python/Python/3/11/3.11.4/Python.Python.3.11.locale.zh-CN.yaml:33]:
```suggestion
ReleaseNotesUrl: https://docs.python.org/release/3.11.4/whatsnew/changelog.html
```

File: manifests/p/Python/Python/3/11/3.11.4/Python.Python.3.11.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+- pyw
+FileExtensions:
+- py
+- pyc
+- pyd
+- pyo
+- pyw
+- pyz
+- pyzw
+ReleaseDate: 2023-04-05
```
Comment (mdanish-kh) [inline manifests/p/Python/Python/3/11/3.11.4/Python.Python.3.11.installer.yaml:21]:
```suggestion
ReleaseDate: 2023-06-06
```

**Review by mdanish-kh — CHANGES_REQUESTED** [review]

**Review by mdanish-kh — APPROVED** [review]

**Pull request merged**